benchmarks/nemo/nemolite2d/manual_versions/psykal_c_opencl/Makefile

```diff
@@ -0,0 +1,91 @@
+# Makefile for manual C OpenCL version of PSyKAl'd NEMOLite2D.
+#
+# This Makefile picks up the compiler to use any flags from
```

Comment:
Good idea. Btw the reason I hardcoded gcc is because I am not sure the `-E -P` flags are standard, and I think all system will have at least some version of gcc to preprocess the kernels